bool is_paused 
---
bool is_paused 